multicluster/cmd/multicluster-controller/controller.go

```diff
@@ -17,14 +17,12 @@ limitations under the License.
 package main
```

Comment:
So, the new binary is called multicluster-controller? Do you have an idea to add "antrea" prefix to the name?

Reply:
oh, I am using the Make target here https://github.com/antrea-io/antrea/pull/2987/files#diff-56964ee07416a48f9f9165deafa2b397c8ddfc3fee80b1618cd3a3eee84a675fR62 to rename it as `antrea-multicluster-controller`, do you suggest to change the folder to `antrea-multicluster-controller` as well?

Reply:
Ah, I was thinking antrea-mc-controller to be shorter, but no strong opinion.